resources/etc/init.d/openhab2

```diff
@@ -12,30 +12,60 @@
 # Author: openHAB <openhab@openhab.org>
 #
 
+# Read configuration variable file if it is present
+if [ -r /etc/default/openhab2 ]; then
+  . /etc/default/openhab2
+fi
+
+if [ -r /etc/default/openhab2_dirs ]; then
+  . /etc/default/openhab2_dirs
+fi
+
+
+if [ -n "${EXTRA_JAVA_OPTS}" ]; then export EXTRA_JAVA_OPTS; fi
+
+# Read and set openHAB variables set in /etc/default/ scripts
+if [ -n "${OPENHAB_HTTP_PORT}" ]; then export OPENHAB_HTTP_PORT; fi
+if [ -n "${OPENHAB_HTTPS_PORT}" ]; then export OPENHAB_HTTPS_PORT; fi
+if [ -n "${OPENHAB_HOME}" ]; then export OPENHAB_HOME; fi
+if [ -n "${OPENHAB_CONF}" ]; then export OPENHAB_CONF; fi
+if [ -n "${OPENHAB_RUNTIME}" ]; then export OPENHAB_RUNTIME; fi
+if [ -n "${OPENHAB_USERDATA}" ]; then export OPENHAB_USERDATA; fi
+if [ -n "${OPENHAB_LOGDIR}" ]; then export OPENHAB_LOGDIR; fi
```

Comment:
You mean I should set defaults here? This would certainly make the init.d script more solid for the default setup, but my aim here was to replicate what happens on systemd, which can't set these dynamically if on of those 5 firs in openhab2_dirs is missing.